### ISSUE TITLE
documentation should mention "open folder" requirement

### ISSUE THREAD

**Issue by Georgeouz** [Gigabyte-Giant/vscode-file-header-comment-helper#11]
This package appears to require an opened folder to work properly. This should be mentioned in the documentation.

**Comment by Gigabyte-Giant** [comment]
You are correct, in order for this extension to work correctly, a project folder must be open in Visual Studio Code.

I will add this information when I update the documentation.

**Comment by Georgeouz** [comment]
I'm not sure I should open another issue for what follows. However, the current documentation would be easier to read if configuration and usage steps were separated. Furthermore, you can safely assume that a user that made it this far is able to install the extension.

## How do I use this?

**Note**: This section makes the assumption that you have a working installation of Visual Studio Code, and are mildly familiar with it. If you aren't, perhaps you should start [here](https://code.visualstudio.com/Docs).

### Define the header template

1. Open your user [or workspace] settings file ([reference](https://code.visualstudio.com/Docs/customization/userandworkspace));
2. Copy the "`fileHeaderCommentHelper.languageConfigs`" property to your settings;
3. Define per-language file-header-comment templates;

```json
// Place your settings in this file to overwrite the default settings
{
    "fileHeaderCommentHelper.languageConfigs": {
        "language_javascript": {
            "template": [
                "/*",
                " * Project: $(projectName)",
                " * File: $(currentFile)",
                " */"
            ]
        }
    }
}
```

### Insert the header into your file

1. Open an existing file (or create a new file) within the project you have just opened;
2. In the "Quick Open" interface, type "`> Insert File-Header-Comment`";

**Comment by Gigabyte-Giant** [comment]
The documentation will be updated with the release of the next version.
Also, I never make assumptions about those whom are using my tools.
